src/server/routes/generators/typescript.ts

```diff
@@ -0,0 +1,22 @@
+import { FastifyInstance } from 'fastify'
+import ejs from 'ejs'
+import { PostgresMeta } from '../../../lib'
+import template from '../../../lib/templates/typescript'
+import { DEFAULT_POOL_CONFIG } from '../../constants'
+
+export default async (fastify: FastifyInstance) => {
+  fastify.get<{
+    Headers: { pg: string }
+  }>('/', async (request, reply) => {
+    const connectionString = request.headers.pg
+    const responseFormat = request.headers.responseFormat || 'text/plain'
+
+    const pgMeta = new PostgresMeta({ ...DEFAULT_POOL_CONFIG, connectionString })
+
+    const { data: tables } = await pgMeta.tables.list({})
+    const { data: functions } = await pgMeta.functions.list({})
+    const definition = ejs.render(template, { tables, functions })
+
+    return reply.header('content-type', responseFormat).send(definition)
```

Comment:
then we can send the data in text/plain, so that the CLI can dump it to disk (or we can display it on the Dashboard etc)

Result:

<img width="544" alt="image" src="https://user-images.githubusercontent.com/10214025/171060777-ad0d85e6-8478-4d43-abef-d8fea78fa5ae.png">

Reply:
Do we need configurable `responseFormat`? Or can we make it `text/plain` always?